# About antiSMASH

antiSMASH allows the rapid genome-wide identification, annotation and analysis
of secondary metabolite biosynthesis gene clusters in bacterial and fungal
genomes. It integrates and cross-links with a large number of in silico
secondary metabolite analysis tools that have been published earlier.

antiSMASH is powered by several open source tools: NCBI BLAST+, HMMer 3, Muscle
3, Glimmer 3, FastTree, TreeGraph 2, Indigo-depict, PySVG and JQuery SVG.

antiSMASH is the product of a collaborative effort between the Department of
Microbial Physiology and Groningen Bioinformatics Centre of the University of
Groningen, the Department of Microbiology of the University of Tübingen, the
Synthetic Biology and Computational and Evolutionary Biology groups at the
Faculty of Life Sciences at the University of Manchester and the Department of
Bioengineering and Therapeutic Sciences at the University of California, San
Francisco.

antiSMASH is supported by the GenBiotics program of the Dutch Technology
Foundation (STW), which is the applied-science division of The Netherlands
Organisation for Scientific Research (NWO) and the Technology Programme of the
Ministry of Economic Affairs (grant STW 10463) as well as the GenBioCom program
of the German Ministry of Education and Research (BMBF) grant 0315585A.

# How to cite

If you have found antiSMASH useful, please cite:

antiSMASH 3.0 — a comprehensive resource for the genome mining of biosynthetic
gene clusters<br>
Tilmann Weber, Kai Blin, Srikanth Duddela, Daniel Krug, Hyun Uk
Kim, Robert Bruccoleri, Sang Yup Lee, Michael A. Fischbach, Rolf Müller,
Wolfgang Wohlleben, Rainer Breitling, Eriko Takano & Marnix H. Medema<br>
Nucleic Acids Research (2015) doi: 10.1093/nar/gkv437.

antiSMASH 2.0 — a versatile platform for genome mining of secondary metabolite
producers.<br>
Kai Blin, Marnix H. Medema, Daniyal Kazempour, Michael A.
Fischbach, Rainer Breitling, Eriko Takano, & Tilmann Weber<br>
Nucleic Acids Research (2013) doi: 10.1093/nar/gkt449.

antiSMASH: Rapid identification, annotation and analysis of secondary
metabolite biosynthesis gene clusters.<br>
Marnix H. Medema, Kai Blin, Peter
Cimermancic, Victor de Jager, Piotr Zakrzewski, Michael A. Fischbach, Tilmann
Weber, Rainer Breitling & Eriko Takano<br>
Nucleic Acids Research (2011) doi:10.1093/nar/gkr466

